tests/sentry/utils/test_redis.py

```diff
@@ -0,0 +1,84 @@
+import functools
+
+import pytest
+
+from sentry.exceptions import InvalidConfiguration
+from sentry.testutils.cases import TestCase
+from sentry.utils.redis import (
+    ClusterManager, _shared_pool, get_cluster_from_options
+)
+
+make_manager = functools.partial(
+    ClusterManager,
+    {
+        'redis.clusters': {
+            'foo': {
+                'hosts': {
+                    0: {'db': 0},
+                },
+            },
+            'bar': {
+                'hosts': {
+                    0: {'db': 0},
+                    1: {'db': 1},
+                }
+            },
+        },
+    },
+)
+
+
+class ClusterManagerTestCase(TestCase):
+    def test_get(self):
+        manager = make_manager()
+        assert manager.get('foo') is manager.get('foo')
+        assert manager.get('foo') is not manager.get('bar')
+        assert manager.get('foo').pool_cls is _shared_pool
```

Comment:
I don't like testing this implementation detail like this — eventually I'd like to make the shared pool a property of the `ClusterManager`, but I don't want to have to expose that as a public attribute (necessary for `make_rb_cluster` compatibility for now.)